exercises/D-strings/exercise.js

```diff
@@ -1,3 +1,5 @@
 // Start by creating a variable `message`
-
+const message = "This is a string."
+var messageType = typeof message;
```

Comment:
try to get into the habit of using `const` instead of `var` for all variables. Then, if the variable _does_ need to change, you can declare it with `let` instead. Try to avoid `var`, as this can cause lots of issues with scoping (https://www.freecodecamp.org/news/var-let-and-const-whats-the-difference/, but don't worry about the details for now)